x-pack/plugins/reporting/server/routes/management/jobs.ts

```diff
@@ -102,29 +141,49 @@ export function registerJobInfoRoutes(reporting: ReportingCore) {
       }
 
       const { docId } = req.params;
-      const {
-        management: { jobTypes = [] },
-      } = await reporting.getLicenseInfo();
 
       const result = await jobsQuery.get(user, docId);
 
-      if (!result) {
-        return res.notFound();
+      const validationResult = await validateJobIsAvailable(result);
+      if (validationResult) {
+        return validationResult;
       }
 
-      const { jobtype: jobType } = result;
+      return res.ok({
+        body: result!,
+        headers: {
+          'content-type': 'application/json',
+        },
+      });
+    })
+  );
+
+  router.get(
+    {
+      path: `${MAIN_ENTRY}/payload/{docId}`,
```

Comment:
Is there any way we can do without adding this new route? It looks quite similar to `${MAIN_ENTRY}/info/{docId}`

Reply:
That's a really good point! I did not realise that this also contained `payload`. I'll remove the new endpoint as it was only being used to provide access to the locatorParams that are already part of info 👍🏻